hbase-server/src/main/java/org/apache/hadoop/hbase/wal/AbstractFSWALProvider.java

```diff
@@ -237,6 +237,8 @@ static void requestLogRoll(final WAL wal) {
   /** File Extension used while splitting an WAL into regions (HBASE-2312) */
   public static final String SPLITTING_EXT = "-splitting";
 
+  public static final String RETRYING_EXT = ".retrying";
```

Comment:
Better add some comments to explain how we use this.

Reply:
sure

Reply:
IIRC the aboce splitting suffix is for the wal directory, and this retrying suffix is for wal file? We'd better mention this difference in the javadoc or comment so later developer will not be confused.

Reply:
done